### ISSUE TITLE
Warning with marginBottom

### ISSUE THREAD

**Issue by borzaka** [thoughtbot/react-native-typescript-styles#9]
![image](https://github.com/user-attachments/assets/86e85711-829f-4543-bcfc-8c44f035f224)

It works fine with:
- `paddingVertical`
- `paddingHorizontal`

My IDE: PhpStorm

**Comment by rakeshpetit** [comment]
On the [Example project](https://github.com/thoughtbot/react-native-typescript-styles-example/tree/main), I modified [some styles](https://github.com/thoughtbot/react-native-typescript-styles-example/blob/main/src/TypographyExample.tsx#L42) based on your screenshot to test and could not reproduce this issue. Based on the style you wrote, `layout`, which is present in https://github.com/thoughtbot/react-native-typescript-styles/blob/main/styles/sizing.ts#L39, this should not throw an ESLint error as the types of `marginBottom` and `layout.x20` are both numbers.

**Comment by borzaka** [comment]
Yes, it's very strage;
![image](https://github.com/user-attachments/assets/0cd2ddd8-8b70-484b-9e5f-40c2c04a4be0)

```javascript
const styles = StyleSheet.create({
  title: {
    marginBottom: 40,
  },
  subtitle: {
    marginBottom: Spacings.x16,
    marginTop: Spacings.x16,
    paddingVertical: Spacings.x16,
    paddingHorizontal: Spacings.x16,
    marginVertical: Spacings.x16,
    marginHorizontal: Spacings.x16,
    paddingTop: Spacings.x16,
    paddingBottom: Spacings.x16,
  },
});
```

I ended up using like this:
`marginBottom: Spacings.x16 as number,`
